tools/circleci/circleci_release.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+"""
+CircleCI pipeline to check if it needs to trigger a release
+"""
+
+import os
+
+import requests
+from requests.structures import CaseInsensitiveDict
+
+from utils.check_no_version_pypi import check_no_version_pypi
+from utils.package_version import get_package_version
+
+PACKAGE_PATHS = (
+    "kedro-datasets/kedro_datasets",
```

Comment:
In theory, we can just use `kedro-datasets` and parse it, but I am not sure if this is a convention. So I just keep the longer path here, not too important anyway.